### ISSUE TITLE
[ feature ] better support for Trung Highway nodes 

### ISSUE THREAD

**Issue by JorgeR81** [chrisgoringe/cg-controller#449]
Some configurations of the `Highway` nodes break upstream "live previews", in the Controller.

( I'm not sure, but I think this was working, at some point ? )

---

This configuration works. 

This `Highway` node it's between the first and second samplers.

![way1](https://github.com/user-attachments/assets/65b3ff65-cb1b-4ac6-bd07-a1b2dc4a48a4)

---

This configuration does not work.

It's functionally the same as the one above, but it allows more flexibility, when building the workflows.

It will work at first but, if we save or switch workflows, the upstream "live previews" will break.  

![way2](https://github.com/user-attachments/assets/a5e9d66f-02b2-408d-bae0-d41167b65894)

**Comment by JorgeR81** [comment]
I've make more tests.

Even with the first configuration, upstream "live previews", can also break if we mute the "in between" groups ( instead of bypass ).

For instance, here is a group, to add a "refiner prompt" ( for the second KSampler ).

Since I use a "bus" node, functionally is the same to mute or bypass this group. 

But muting, breaks upstream "live previews", in the Controller.

---

![bp](https://github.com/user-attachments/assets/6f2570f5-7dd0-4441-b3d0-df0e70f26944)

**Comment by JorgeR81** [comment]
I've made some more tests.

For the upstream "live previews" to work, we need some other connection between the Highway nodes, like this:

![xx1](https://github.com/user-attachments/assets/931396ac-4301-4adb-a7e4-a602f0a0d0f2)

**Comment by JorgeR81** [comment]
I tested the the `Context (rgthree)` node an it has the same issue.

For the "live preview" to work I need to connect the "latent" between the nodes.

--

**Example**

This group is between the first and second KSampler. It's used to "refine" the prompt.
The "latent" connections don't do anything to the image. They are there just to fix the upstream "live previews".

![cx](https://github.com/user-attachments/assets/7e47bd77-2d50-4028-923b-6ed2c08d11d3)

**Comment by chrisgoringe** [comment]
Right. It only follows images and latents upstream for efficiency (and
avoiding false positives).

I can look into following some of the common pipes as well

**Comment by JorgeR81** [comment]
> I can look into following some of the common pipes as well

Yeah, that would be a good idea.

But `was-node-suite` and `Impact-Pack` probably don't need this, because they don't support latent wires.
So we need to run separate latent wires anyways.

https://github.com/WASasquatch/was-node-suite-comfyui
https://github.com/ltdrdata/ComfyUI-Impact-Pack

![b0](https://github.com/user-attachments/assets/3dc986d4-a5f1-4c40-96e4-93a632245aa9)

----

I know about another configurable bus node that supports latent wires, but I never tried. 

https://github.com/MaraScott/ComfyUI_MaraScott_Nodes?tab=readme-ov-file#how-to-use
